tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -18,24 +18,81 @@
 from tests.common.utils import retrieve_latest_ami
 
 
-@pytest.mark.skip(reason="Temporarily disable this test")
 @pytest.mark.dimensions("eu-west-1", "c5.xlarge", "alinux", "*")
 @pytest.mark.dimensions("us-west-1", "c5.xlarge", "alinux2", "*")
 @pytest.mark.dimensions("us-west-2", "c5.xlarge", "centos7", "*")
 @pytest.mark.dimensions("eu-west-2", "c5.xlarge", "ubuntu1604", "*")
+# fixme ubuntu1804 have issue
 @pytest.mark.dimensions("us-east-1", "c5.xlarge", "ubuntu1804", "*")
 @pytest.mark.dimensions("us-gov-east-1", "c5.xlarge", "ubuntu1604", "*")
 @pytest.mark.dimensions("us-gov-west-1", "c5.xlarge", "ubuntu1804", "*")
 @pytest.mark.dimensions("cn-northwest-1", "c4.xlarge", "alinux2", "*")
-@pytest.mark.dimensions("us-east-1", "m6g.xlarge", "ubuntu1804", "*")
-@pytest.mark.dimensions("eu-west-1", "m6g.xlarge", "alinux2", "*")
+# Temporarily skip official ami(vanilla ami)
```

Comment:
I think we are skipping the test on remarkable ami for ARM, because there is no AMI at the moment. If this is correct we could remove the mark below

Reply:
Done